main.py

```
@@ -604,0 +1,1133 @@
+                output_format=validated_format,
+                save_chunks=save_chunks,
+                chunk_dir=chunk_dir,
+                chunk_format=chunk_format,
+                skip_cache=skip_cache,
+            )
+
+        logger.info(
+            f"Process completed successfully. Output saved in {validated_format} format."
+        )
+    except Exception as e:
+        logger.error(f"An error occurred during the process: {e}", exc_info=True)
```

Ruff (0.8.2)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)